ansible/roles/stack-sunbird/templates/ml-survey-service.env

```diff
@@ -0,0 +1,151 @@
+
+{{ml_survey_cloud_config}}
+
+# Mongodb connection url
+MONGODB_URL={{ml_survey_mongodb_url}}
+
+# Base url of the sunbird enviornment
+USER_SERVICE_URL={{ml_survey_user_service_URL}}
+
+# Current name for mongodb database 
+DB={{ml_survey_db_name|default("sl"+env)}}
+
+# Secure token access Admin related API's
+# The value of internal access token can be generated by command - "openssl rand -hex 10"
+INTERNAL_ACCESS_TOKEN={{ml_survey_internal_access_token}}
+
+# CSV Config
+# Create the respective directory as provided in the path below (Path to folder where reports will be stored)
+CSV_REPORTS_PATH={{ml_survey_csv_reports_path | default("reports")}}
+
+# Kafka Configuration
+# Flag which define where communication with kafka is on or off 
+KAFKA_COMMUNICATIONS_ON_OFF={{ml_survey_kafka_communications_on_off | default("ON")}}
+
+# IP address of kafka server with port without HTTP
+KAFKA_URL={{ml_survey_kafka_url}} 
+
+# Kafka topic name for pushing completed sumbmission
+COMPLETED_SUBMISSION_TOPIC={{ml_survey_completed_submission_topic | default (env+".sl.submission.raw")}}
+
+# Kafka topic name for pushing incompleted submission
+INCOMPLETE_SUBMISSION_TOPIC={{ml_survey_incomplete_submission_topic | default (env+".sl.incomplete.submission.raw")}}
+
+# Kafka topic name for pushing submissions for which rating has to be done
+SUBMISSION_RATING_QUEUE_TOPIC={{ml_survey_submission_rating_queue_topic | default (env+".sl.submission.rating.raw")}}
+
+# Kafka topic name for pushing completed observation submission
+COMPLETED_OBSERVATION_SUBMISSION_TOPIC={{ml_survey_completed_observation_submission_topic | default (env+".sl.observation.raw")}}
+
+# Kafka topic name for pushing incomplete observation submission
+INCOMPLETE_OBSERVATION_SUBMISSION_TOPIC={{ml_survey_incomplete_observation_submission_topic | default (env+".sl.incomplete.observation.raw")}}
+
+# Kafka topic name for pushing notification messages
+NOTIFICATIONS_TOPIC={{ml_survey_notifications_topic | default (env+".sl.notification.raw")}})
+
+# Kafka topic name for completed survey submission
+COMPLETED_SURVEY_SUBMISSION_TOPIC={{ml_survey_completed_survey_submission_topic | default (env+".sl.survey.raw")}}
+
+# Kafka topic name for incomplete survey submission
+INCOMPLETE_SURVEY_SUBMISSION_TOPIC={{ml_survey_incomplete_survey_submission_topic | default (env+".sl.incomplete.observation.raw")}}
+
+# Kafka consumer group for ML Survey Service
+KAFKA_GROUP_ID={{ml_survey_kafka_group_id | default (env+"samiksha")}}
+
+#  Kafka topic name for pushing project submission related data
+IMPROVEMENT_PROJECT_SUBMISSION_TOPIC={{ml_survey_improvement_project_submission_topic | default(env+".sl.improvement.project.submission")}}
+
+# Elastic search configurations 
+# Elastic search enable/disable flag E.g. ON/OFF
+ELASTICSEARCH_COMMUNICATIONS_ON_OFF={{ml_survey_elasticsearch_communications_on_off | default("ON")}}
+
+# Elastic search host url
+ELASTICSEARCH_HOST_URL={{ml_survey_elasticsearch_host_url}}
```

Comment:
Where are we planning to define these host variables? We should not be defining them in private repo, instead please add a `groups['']` placeholder in `all.yml`. Please do the same for other url's like mongo, kafka etc.

Reply:
done